LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -176,6 +176,10 @@
  */
 @property (nonatomic, strong) NSString  *keychainTimerDurationUsername;
 /**
+ @brief The string to be used as username for the "isSimple" in the Keychain.
+ */
+@property (nonatomic, strong) NSString  *keychainIsSimpleUsername;
```

Comment:
Could we change the name to the more descriptive naming, like for user defaults, `keychainPasscodeIsSimpleUsername`?